docs/src/jsx/utils/WorldviewCodeEditor.js

```diff
@@ -113,7 +114,7 @@ export const scope = {
   Text,
   GLTFScene,
   withPose,
-
+  getRayFromClick,
```

Comment:
This change seems unrelated to the rest of this PR, did you mean to commit this?

Reply:
I started out hoping to build the dragging example, but realized it's better to keep it in a separate PR. Overall it's good to have this function in the global scope of code playground just people can use it.